edgelet/iotedged/src/lib.rs

```diff
@@ -2074,11 +2077,18 @@ where
     );
 
     env.insert(WORKLOAD_URI_KEY.to_string(), workload_uri);
+
     #[cfg(feature = "runtime-docker")]
-    env.insert(
-        WORKLOAD_LISTEN_MNT_URI.to_string(),
-        Listen::workload_mnt_uri(&home_dir),
-    );
+    if env::var(IOTEDGE_WORKLOAD_SOCKETS_ENABLE)
+        .as_deref()
+        .map_or(false, |value| value.eq_ignore_ascii_case("true"))
+    {
+        env.insert(
+            WORKLOAD_LISTEN_MNT_URI.to_string(),
+            Listen::workload_mnt_uri(&home_dir),
+        );
+    }
```

Comment:
We discussed this in shiproom today, and the conclusion was that adding this flag might make the situation more confusing for customers, so let's just go with the fix in EdgeAgent for now, and not add the flag.

Reply:
Removed!